cluster/os-3.9.0-alpha.4/provider.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+prefix=kubevirt-os-3.9.0-alpha.4
+
+function _main_ip() {
+    echo 127.0.0.1
+}
+
+_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock rmohr/cli:latest'
```

Comment:
instead of using latest, use a specific version of the cli container image.

Reply:
fixed